src/App.js

```diff
@@ -1,15 +1,36 @@
-import React from 'react'
+import React, { useState } from 'react'
 import Header from './components/Header'
 import AddTodo from './components/AddTodo'
 import Todo from './components/Todo'
 import Footer from './components/Footer'
 import './App.css'
-export default function App (){
+import uuid from 'react-uuid'
+export default function App() {
+    const [todoItems, setTodoItems] = useState([])
+    const [todoInput, setTodoInput] = useState("")
+
+    function handleTodoInput(e) {
+        const newValue = e.target.value
+        console.log(newValue)
+        setTodoInput(newValue)
+    }
+
+    function addItem(e) {
+        e.preventDefault()
+        const newItem = { id: uuid(), value: todoInput, isDone: false }
+        setTodoItems(prevItems => [...prevItems, newItem])
+        console.log(todoItems)
+        setTodoInput("")
```

Comment:
All these functionality should be moved into the `AddTodo` component.

`App.js` doesn't need to know how `AddTodo` is handling the addition of the todo items, it should only concern itself with holding all the todo items (for now, of course)